planetarium/NineChronicles
# Implement equipment level vfx. (WIP)

## Applied features
* [ ] Applies VFX in weapon for particular equipment level.

## Required Reviewers
Namyujeong and @planetarium/nine-chronicles-engineers 

## Test Method
* Equip Surtr's Sword (Fire).
* Try equipping/unequipping or changing to another weapon to check the VFX's lifecycle is handling properly.
* Enter any stage and check the VFX is connected properly with player spine object's hand.

## Screenshots
![GIF 2021-02-24 오후 9-04-45](https://user-images.githubusercontent.com/26648527/108998267-0301c200-76e4-11eb-8251-36e30adf4d1f.gif)



┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1141562434100787/1200503068471083) by [Unito](https://www.unito.io)
